src/main/groovy/wooga/gradle/unity/UnityPluginConventions.groovy

```diff
@@ -73,7 +74,19 @@ class UnityPluginConventions implements PlatformUtilsImpl {
     /**
      * The path to the Unity Editor executable
      */
-    static final PropertyLookup unityPath = new PropertyLookup(["UNITY_UNITY_PATH", "UNITY_PATH"], "unity.unityPath", { getPlatformUnityPath().absolutePath })
+    static final PropertyLookup unityPath = new PropertyLookup(["UNITY_UNITY_PATH", "UNITY_PATH"], "unity.unityPath", { getPlatformUnityPath().absolutePath })    /**
+     /**
+     * The path to the Unity Editor root folder (Unity or Unity.app in the default installation).
+     */
+    static final PropertyLookup unityRoot = new PropertyLookup(["UNITY_UNITY_ROOT", "UNITY_ROOT"], "unity.unityRoot", null)
```

Comment:
I would disambiguate here between this and unityPath. Perhaps calling this one unityRootDirectoryPath?